filters/remove_frequent_patterns.txt

```diff
@@ -0,0 +1,9 @@
+# Starting text in a lot of rows in OpenSubtitles, not everytime in both source and targets
+^ -	
```

Comment:
You can't really see it easily here, but the lines with the patterns end with a tab.